README.md

```diff
@@ -383,7 +383,6 @@ Login to azure using the az cli. Packer will use the session setup by the az cli
 packer build \
     -var 'spel_identifier=unique-project-id' \
     -var 'spel_version=0.0.1' \
-    -var 'spel_disablefips=true' \
```

Comment:
Defaulting to having FIPS-mode enabled in Azure VM-templates?

Reply:
Well, _overriding_ the default. But also, this particular variable was renamed in switch to the hcl syntax, to `amigen_fips_disable`, which is still specified in the example command, just a couple lines down...